### DISCUSSION TITLE
Future crop yield question

### DISCUSSION THREAD

**Discussion by whq552800** [iiasa/CWatM#153] [Q&A]
I am attempting to use CWatM to simulate future crop yields. I have followed steps from a YouTube tutorial ’https://www.youtube.com/watch?v=YvdVTB4wtTA&t=397s‘, and I am using mapspam data. To simplify the computation, I applied a uniform date globally. 
![image](https://github.com/user-attachments/assets/addad10b-82ba-4b69-a9a0-58595cae7bf2)

The resolution is set at 30 arcminutes. The following outputs were generated: OUT_MAP_AnnualTot = actTransTotal_month_nonIrr, Yield_Irr, Yield_nonIrr, crop_eta_nonirr, crop_eta_irr, ratio_a_p_nonIrr, ratio_a_p_Irr, crop_pet_irr, crop_Precipitation, crop_Tavg.
Some of the variables in it are edited by me according to my personal needs.
To obtain results for each crop, I modified the code for internal model output in the “output.py” file. Details of the modification can be found in the attached image.
![image](https://github.com/user-attachments/assets/2073fde7-2125-4291-b6c8-1e81c492fb5f)

Based on existing research, maize (corn) is the crop most responsive to future climate changes among the four crops studied. Additionally, the area of irrigated maize in the mapspam data is very small, so I focused on analyzing rainfed maize. However, the results I obtained show a yield decline of approximately 6%(Scenario:SSP585, variable: Yield_nonIrr_annualtot_MAIZ, I used the mean of the variables in the last 30 years (2070-2100) to compare the changes in the first 10 years (2020-2030)), which is much lower than the 30% decline reported in published papers. I am unsure where the issue lies.
![image](https://github.com/user-attachments/assets/871f44e3-f575-4eb5-a0bd-64517c0f8011)
The picture below shows the interannual variation of rain-fed maize.
![image](https://github.com/user-attachments/assets/c0631e42-f60e-4eae-8077-cd2bd72bd74d)

P.S.: The mapspam dataset includes crop yield data, and I am considering calculating future yields by taking the ratio of the annual relative crop yield (Yr) to the Yr from the most recent year (2019) obtained using GWSP3-W5E5, and then multiplying this ratio by the yield values in the mapspam dataset. Would this approach be appropriate?
![image](https://github.com/user-attachments/assets/96816c61-9a2d-479d-a796-b1fbfd6b3a6e)

**Comment by whq552800** [comment]
Below is the data related to rain-fed maize under the SSP585 scenario under CNRM-CM6-1
![image](https://github.com/user-attachments/assets/63a04599-1b5c-4a63-bd99-199e24e0265a)
![image](https://github.com/user-attachments/assets/78d1ac82-bf8a-4878-b333-87a6efb7187b)
![image](https://github.com/user-attachments/assets/e3f2847c-1d35-479a-9b06-c1290b2d5111)

**Reply by dof1985** [reply]
Hi @whq552800 

It is nice to see your work and results. Though it is hard to say where exactly the problem lies, I would point out two gaps I can think of: 
(A) I miss a lot of information regarding the studies you cite showing a 30% decrease. What are the models used for this assessment? do they only account for yield loss due to water deficit (as in CWatM) or also due to lack of nutrients, heat stress, etc.? Also how uncertain are these numbers? which climate models and scenarios do they use?
(B) to what extent the Kc, Ky factors you have used (one global set of parameters) affect your results? some sensitivity analysis would be nice. Finally, can you extract these parameters (including planting dates and growing stages length) from the studies you have cited? 

Best
Dor Fridman

**Reply by whq552800** [reply]
Dear Dr. Fridman,

Thank you for your prompt reply.

I would like to provide some references I have consulted in my research:

[1] Jägermeyr, J., Müller, C., Ruane, A.C., et al. "Climate impacts on global agriculture emerge earlier in new generation of climate and crop models." Nature Food, 2, 873–885 (2021). 
![image](https://github.com/user-attachments/assets/c29b61ce-7727-4e23-aec5-e8dfb9a64269)

This study utilizes agricultural models to simulate future crop yields using CMIP6 climate data from the ISIMIP3b rounds. It reports a projected 25% decrease in maize yield under the SSP585 scenario.

[2] Challinor, A., Watson, J., Lobell, D., et al. "A meta-analysis of crop yield under climate change and adaptation." Nature Climate Change, 4, 287–291 (2014). 
![image](https://github.com/user-attachments/assets/92f6c64b-0bc5-4918-9ded-9e3bd5aef715)
The gap 2,
This meta-analysis indicates a significant decrease in temperate maize yield under scenarios such as A1B.
[3] Hasegawa, T., Wakatsuki, H., Ju, H., et al. "A global dataset for the projected impacts of climate change on four major crops." Scientific Data, 9, 58 (2022). 
![image](https://github.com/user-attachments/assets/4528c9af-0ebe-4821-b74d-4eb31a0dcfa5)

[4] This study provides a global analysis of projected impacts on crop yields based on data from the past four decades.
Schlenker, W., Roberts, M.J. "Nonlinear Temperature Effects Indicate Severe Damages to U.S. Crop Yields under Climate Change." Proceedings of the National Academy of Sciences U.S.A., 106 (37), 15594–15598 (2009). 

In this study,Holding current growing regions fixed, area-weighted average yields are predicted to decrease by 30 – 46% before the end of the century under the slowest (B1) warming scenario and decrease by 63– 82% under the most rapid warming scenario (A1FI) under the Hadley III model.

It is important to note that these studies generally use crop calendars, with different planting and maturity dates for each location. For the United States and China, maize planting periods typically range from April to October, which aligns with the period I have selected. However, there are still significant discrepancies with other research findings.
Additionally, I have downloaded output results from ISIMIP related to Reference [1] and compared data from three models on yield, actual evapotranspiration, and potential evapotranspiration. I observed significant discrepancies between my data and the actual evapotranspiration data provided by one of these models. Below is an example of the actual evapotranspiration from this model.
![image](https://github.com/user-attachments/assets/0d786f6f-ac8b-4ba9-af1e-293d866e13d4)
![image](https://github.com/user-attachments/assets/324796f7-7780-47e0-9c79-6fcac37676ad)
Furthermore, I am currently working on sensitivity analysis for KC and KY. I plan to share the results in a few days.
![image](https://github.com/user-attachments/assets/6fbbc96b-9f6e-4614-8656-34d32acab661)

Thank you again for your assistance.

Best regards,
Wang Haiqing

**Reply by whq552800** [reply]
I have used the following formula to represent the sensitivity of the factors:
![image](https://github.com/user-attachments/assets/303a6f0d-9abb-430d-aad2-e1879998a86b)
The magnitude of the value indicates sensitivity, with positive numbers showing a decrease in relative yield change and negative numbers indicating an increase in relative yield. Below is the average across different ratios:
![image](https://github.com/user-attachments/assets/6bf66650-c933-4c37-9e4e-e600bf933275)
There is a significant relationship between yield and Ky, but the relationship with Kc is more complex.

**Reply by dof1985** [reply]
Dear @whq552800 

Thank you for your elaborated response. It looks like you have done a through exploration of the literature, well done. Unfortunately, I could not go in depth into all the citation you have presented. What I do see (particularly in the first two papers) is quite large uncertainty and disagreement between the models, showing a decrease for maize (relative to the the present) that ranges between 10%-25% (second one, and can be much below 30% for the first paper.

One major point I have noticed. I suspect you may mis-use CWatM crops' functionality. The module primary aim is to provide a better estimation of plant water requirements affecting soil moisture and irrigation demand. 
The calculated yield loss (e.g., with Ky) is the loss in % relative to a reference yield (which is a function of cultivar, management, etc.).
For example, if the reference yield (without water stress, e.g., under full irrigation) is 10 ton/ha and you have a 10% loss due to water deficit - the actual yield would be 10 x (1- 0.1) = 9 ton/ha. 

To get the change in future yield relative to the past you would have to obtain potential yields under full irrigation (e.g., from GAEZ: https://gaez.fao.org/) for the past and future, calculate actual yield for both period and only than calculate the relative change, as: future_yield / past_yield - 1.

Hope this would help you somehow.

**Reply by whq552800** [reply]
Dear Dor Fridman,
I would like to express my sincere gratitude once again for your guidance. 
I have revisited the mechanism of CWatM and related topics, adjusted the direction of my research, and recalculated the results based on your suggestions. The issue has been resolved. Please forgive my limited knowledge. Below are the results I have obtained:
![image](https://github.com/user-attachments/assets/9d0f9bcf-9a57-462e-b676-855748ec1978)
Best regards,
Wang Haiqing

**Reply by dof1985** [reply]
Dear @whq552800 

No worries. We are happy to help CWatM's users community. The model is complex and constantly improving, thus it may be challenging to keep track. 
I am looking forward to see more from your work. 
Please feel free to share any additional questions or interesting findings.

Best
Dor
